docs/install/1-install-server.rst

```diff
@@ -376,34 +376,14 @@ Replace <password> in the above-mentioned snippet with the password of the inman
 connect to the local server and uses the database inmanta. See the :inmanta.config:group:`database` section in the
 config file for other options.
 
-.. _configure_server_step_5:
-
-Step 6: Set the server address
-------------------------------
-
-When virtual machines are started by this server that install the inmanta agent, the correct
-:inmanta.config:option:`server.server-address` needs to be
-configured. This address is used to create the correct boot script for the virtual machine.
-
-Set this value to the hostname or IP address that other systems use to connect to the server
-in the configuration file stored at ``/etc/inmanta/inmanta.d/server.cfg``.
-
-.. code-block:: text
-
-  [server]
-  server-address=<server-ip-address-or-hostname>
-
-.. note:: If you deploy configuration models that modify resolver configuration it is recommended to use the IP address instead
-  of the hostname.
-
```

Comment:
This cannot be dropped, can it? The scheduler needs this option to know how to connect back to the server.

Reply:
oof, right